### PR TITLE
Added code for bit division in C++ language

### PR DESCRIPTION
**Pull request by rbjoshi1309**
**Fixes issue:** #4743 

**Changes:**

Added C++ version for bit division in following folder :
cosmos/code/bit_manipulation/src/bit_division

### CODE REVIEW

File: code/bit_manipulation/src/addition_using_bits/addition_using_bits.c [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*
+Problem Statement : 
+Add two numbers using bitwise operators
+(i.e without using arithmetic operators)
+*/
+
+#include<stdio.h>
```
Comment (arnavb) [inline code/bit_manipulation/src/addition_using_bits/addition_using_bits.c]:
```suggestion
#include <stdio.h>
```

File: code/bit_manipulation/src/bit_division/bitdivision.cpp [hunk 2]
```diff
@@ -0,0 +1,62 @@
+/*
```
Comment (arnavb) [inline code/bit_manipulation/src/bit_division/bitdivision.cpp]: Please call the file the same thing as the folder.

File: code/bit_manipulation/src/bit_division/bitdivision.cpp [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Problem Statement : 
+Divide two numbers using bitwise operators
+(i.e without using arithmetic operators)
+*/
+
+#include <iostream>
+using namespace std;
```
Comment (arnavb) [inline code/bit_manipulation/src/bit_division/bitdivision.cpp]: Remove this.

File: code/bit_manipulation/src/bit_division/bitdivision.cpp [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        denominator >>= 1;
+    }
+
+    return answer;
+}
+
+int main() 
+{
+    int numerator, denominator;
+    // numerator -> represents divident
```
Comment (arnavb) [inline code/bit_manipulation/src/bit_division/bitdivision.cpp]:
```suggestion
    // numerator -> represents dividend
```

File: code/bit_manipulation/src/bit_division/bitdivision.cpp (hunk 3 above)
Comment (rbjoshi1309) [reply code/bit_manipulation/src/bit_division/bitdivision.cpp]: Do you mean, removal of "using namespace std" ??

File: code/bit_manipulation/src/addition_using_bits/addition_using_bits.c (hunk 1 above)
Comment (rbjoshi1309) [reply code/bit_manipulation/src/addition_using_bits/addition_using_bits.c]: The two files addition_using_bits.c and addition_using_bits.java got added by mistake.
These files are already merged in other PR #4727.
Should I close this PR and raise a new one for file bit_division.cpp or this PR should be fine ??

File: code/bit_manipulation/src/bit_division/bitdivision.cpp (hunk 4 above)
Comment (rbjoshi1309) [reply code/bit_manipulation/src/bit_division/bitdivision.cpp]: Done.

File: code/bit_manipulation/src/bit_division/bitdivision.cpp (hunk 2 above)
Comment (rbjoshi1309) [reply code/bit_manipulation/src/bit_division/bitdivision.cpp]: Done

**Pull request merged**